### DISCUSSION TITLE
How replace img in this code

### DISCUSSION THREAD

**Discussion by billpi1** [tailwindlabs/tailwindcss#13166] [Help]
Hello
Please look at below code , i want to replace bg color to background image . How must be code:

```css
ClassName="h-full w-full bg-[#ffdd780]"
```

**Comment by wongjn** [comment]
```html
<element className="h-full w-full bg-[url(/path/to/image)]">
```
[See the documentation for details](https://tailwindcss.com/docs/background-image#arbitrary-values).
